README.md

```diff
@@ -62,16 +68,20 @@ npm update -g generator-jhipster-nav-element
 yarn global upgrade generator-jhipster-nav-element
 ```
 
+# Contributors ✨
 
-# License
+Thanks goes to these wonderful people:
+
+<table><tr><td align="center"><a href="https://github.com/vivekmore"><img src="https://avatars1.githubusercontent.com/u/2099329?s=400&v=4" width="100px;" alt="Vivek More (founder)"/><br/><sub><b>Vivek More</b><br/><b>(founder)</b></sub></a></td><td align="center"><a href="https://github.com/amanganiello90"><img src="https://avatars3.githubusercontent.com/u/20536757?s=400&v=4" width="100px;" alt="Angelo Manganiello"/><br /><sub><b>Angelo Manganiello</b></sub></a></td></tr></table>
 
-Apache-2.0 © [Vivek More]
+# License
 
+Apache-2.0 © [Vivek More](https://github.com/vivekmore)
 
 [npm-image]: https://img.shields.io/npm/v/generator-jhipster-nav-element.svg
 [npm-url]: https://npmjs.org/package/generator-jhipster-nav-element
-[travis-image]: https://travis-ci.org/vivekmore/generator-jhipster-nav-element.svg?branch=master
-[travis-url]: https://travis-ci.org/vivekmore/generator-jhipster-nav-element
+[github-actions-generator-ci-image]: https://github.com/jhipster/jhipster-quarkus/workflows/Generator%20CI/badge.svg
```

Comment:
I don't want to lose the travis build info, please retain that

Reply:
ok